app/models/rook.rb

```diff
@@ -1,2 +1,8 @@
 class Rook < ChessPiece
+  def valid_move?(destination_x, destination_y)
+    return false if out_of_bounds?(destination_x, destination_y)
+    return false if diagonal_move?(destination_x, destination_y)
+    return false if obstructed?(destination_x, destination_y)
+    horizontal_move?(destination_x, destination_y) || vertical_move?(destination_x, destination_y) ? true : false
```

Comment:
You should never have to use a ternary to return true and false. In this case you can just remove the ternary and get the exact same result.